src/style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background-color: black;
+  --red-border: 1px solid #e95d49;
+}
+
+.block {
+  padding: 20px;
+}
+
+.blue-box {
+  width: 350px;
+  background-color: #308dc0;
+  border: var(--red-border);
+  padding: 10px;
```

Comment:
You have border and padding styles same for three classes.
Create common class for these styles.